benchmark/utils/DriverUtils.swift

```diff
@@ -268,7 +298,11 @@ func runBench(_ name: String, _ fn: (Int) -> Void, _ c: TestConfig) -> BenchResu
     var elapsed_time : UInt64 = 0
     if c.fixedNumIters == 0 {
       elapsed_time = sampler.run(name, fn: fn, num_iters: 1)
-      scale = UInt(time_per_sample / elapsed_time)
+      if elapsed_time > 0 {
```

Comment:
Fixed in https://github.com/apple/swift/pull/7810/commits/9eeac18487b3018d8d94fb2f0a517c9fa93ce730